server/app/services/geo/AddressSuggestionGroup.java

```diff
@@ -0,0 +1,31 @@
+package services.geo;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Represents address candidates returned from Esri's findAddressCandidates
+ *
+ * <p>See {@link EsriClient} for details on getAddressCandidates
+ */
+@AutoValue
+public abstract class AddressSuggestionGroup {
+  public static Builder builder() {
+    return new AutoValue_AddressSuggestionGroup.Builder();
+  }
+
+  /** returns a well-known ID for ArcGIS coordinate systems, used for spatial reference */
+  public abstract int getWellKnownId();
+
+  /** returns a list of address suggestions sorted by their score (best match) */
+  public abstract ImmutableList<AddressSuggestion> getAddressSuggestionGroup();
```

Comment:
suggestion (non-blocking): This class is the group

```suggestion
  public abstract ImmutableList<AddressSuggestion> getAddressSuggestions();
```